src/zope/component/tests/test__declaration.py

```diff
@@ -80,22 +80,11 @@ def _run_generated_code(self, code, globs, locs,
                             fails_under_py3k=True,
                            ):
         import warnings
-        #from zope.component._compat import PYTHON3
-        PYTHON3 = False
```

Comment:
I don't think that's actually necessary. The `fails_under_py3k` argument can just be removed now...all the tests pass under both versions.